pom.xml

```diff
@@ -1814,6 +1814,29 @@ flexible messaging model and an intuitive client API.</description>
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <maven.compiler.release>${maven.compiler.target}</maven.compiler.release>
+        <!-- required for running tests on JDK17+ -->
+        <test.additional.args> --add-opens java.base/java.lang=ALL-UNNAMED </test.additional.args>
```

Comment:
What are the components that are failing with Java17? We should probably fixed these instead of just enabling this here.